src/System.Private.Xml/src/System/Xml/Resolvers/XmlXapResolver.cs

```diff
@@ -1,79 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Xml {
-    using System;
-    using System.IO;
-    using System.Diagnostics;
-
-    //
-    // Application stream resolver interface. See XmlXapResolver.s_appStreamResolver for more comments.
-    //
-    [System.Runtime.CompilerServices.FriendAccessAllowed] // used from System.Windows.dll
-    internal interface IApplicationResourceStreamResolver {
-        Stream GetApplicationResourceStream(Uri relativeUri);
-    }
-
-    /// <summary>
-    /// Resolves resources from the application package = xap
-    /// Only applicable to Silverlight
-    /// </summary>
-    public class XmlXapResolver : XmlResolver {
```

Comment:
[This type](https://referencesource.microsoft.com/#System.Xml/System/Xml/XmlXapResolver.cs,94fe2116c1bfb8a8) was labeled as `Obsolete` and it wasn't intended to be exposed for NET Core